### ISSUE TITLE
Playlist autoplay next video - possible to default off?

### ISSUE THREAD

**Issue by m2pt5** [YePpHa/YouTubeCenter#503]
Every time I open a video from a playlist (including my Youtube favorites) I have to manually turn off the autoplay so it doesn't jump to the next video when the current one ends. Would it be possible to add an option to force this to default to off? (If it already exists, I can't find it in the current dev version.)

**Comment by ParthThakur** [comment]
@clst The script won't work anymore. :(
If you guys have a solution...

**Comment by clst** [comment]
Yes, my script is broken, but YouTubeCenter worked here yesterday.
I don't know if Youtube broke it again though.
